src/core/textures/Texture.js

```diff
@@ -258,6 +280,24 @@ export default class Texture extends EventEmitter
     }
 
     /**
+     * returns uvs
+     * @member PIXI.TextureUvs#
+     * @memberof PIXI.Texture#
+     */
+    get uvs()
+    {
+        return this._uvs;
+    }
+
+    /**
+     * Updates uvs accordingly to the texture
+     */
+    updateUvs()
+    {
+        this._updateUvs();
+    }
+
+    /**
      * Helper function that creates a Texture object from the given image url.
      * If the image is not in the texture cache it will be  created and loaded.
      *
```

Comment:
Cool, maybe not for this PR but I guess we could remove the ```_``` prefix from the properties to indicate they are now public.